eos-core-depends

```diff
@@ -196,6 +195,7 @@ podman-toolbox
 # For modem support
 ppp
 printer-driver-all-enforce
+printer-driver-hpcups
```

Comment:
I don't think we need to add this, the printer driver all package above should depend on it.